CHANGELOG.md

```diff
@@ -1,36 +1,42 @@
 # Changelog
 
+## v12.0.0
+
+* Support for a CloudFront Distribution has been deprecated.
+* Support for Classic WAF has been deprecated: only WAFv2 associations will be made moving forward.
+* Support for Default VPC and Subnets has been deprecated: values must be provided to the module.
+* Security Groups are no longer provisioned and must be provided to the module.
```

Comment:
👋 I'm curious why this change was made.  Just too much of a pain to support?

Reply:
:wave: you nailed it, and the agency does not use the CloudFront feature or Default subnets. So, ripped it out, and were trying to push the security settings onto the engineer using the module.